strax/run_selection.py

```diff
@@ -260,27 +263,38 @@ def define_run(self: strax.Context,
                data: ty.Union[np.ndarray, pd.DataFrame, dict],
```

Comment:
Would you mind adding a docstring to this function? Perhaps for data it might be worth adding what data it is or rename the variable as it's very generic